source/_posts/2019-01-23-lovelace-released.markdown

```diff
@@ -32,6 +32,8 @@ For a deep dive into all the new features, check out the [latest episode of the
 
 For an overview of all the different features, check out the [Lovelace documentation](/lovelace/).
 
+If you've not used Lovelace before, then a new default configuration will be generated, based off of your existing configuration. Don't remove any groups until after you've transitioned.
```

Comment:
Isn't this only relevant if you are using one of the auto-generated modes? Maybe worth clarifying.

Reply:
Yes, it is _only_ for auto-generated mode. I did not see this PR and had added this note https://www.home-assistant.io/blog/2019/01/23/lovelace-released/#migrating-to-lovelace